server/athenian/api/align/queries/metrics.py

```diff
@@ -194,6 +203,62 @@ def _parse_time_interval(params: Mapping[str, Any]) -> Intervals:
     return goal_dates_to_datetimes(date_from, date_to)
 
 
+def _parse_requested_metrics(params: Mapping[str, Any]) -> Sequence[_RequestedMetric]:
+    metrics = params.get(MetricParamsFields.metrics)
+    metrics_w_params = params.get(MetricParamsFields.metricsWithParams)
+
+    if not (bool(metrics) ^ bool(metrics_w_params)):
```

Comment:
```suggestion
    if bool(metrics) == bool(metrics_w_params):
```
xor is `!=`. `not (a != b)` is equivalent to `a == b`. You probably want ` if bool(metrics) != bool(metrics_w_params):` here, though.

Reply:
Yes I want not xor, do we prefer `bool(metrics) == bool(metrics_w_params)` as style ?

Reply:
Yes, please, let's use `==` and `!=` everywhere. `^` is reserved for bitwise manipulation in numpy. Let's simplify the life of future juniors and middles.